virtool/db/mongo.py

```diff
@@ -1,34 +1,34 @@
 import sys
 from logging import getLogger
-from typing import Callable, List
 
 from motor.motor_asyncio import AsyncIOMotorClient
 from pymongo import ASCENDING, DESCENDING
 from pymongo.errors import ServerSelectionTimeoutError
 from semver import VersionInfo
 
-from virtool.config.cls import Config
-from virtool.db.core import DB
 
 MINIMUM_MONGO_VERSION = "3.6.0"
 
 logger = getLogger("mongo")
 
 
 async def connect(
-    config: Config, enqueue_change: Callable[[str, str, List[str]], None]
+    db_connection_string: str, db_name: str
 ):
```

Comment:
Annotate with appropriate return type using `->`.

See [this](https://motor.readthedocs.io/en/stable/api-asyncio/asyncio_motor_client.html) to understand what is returned by `db_client[db_name]`.